quiche/src/h3/qpack/mod.rs

```diff
@@ -143,20 +151,73 @@ mod tests {
             crate::h3::HeaderRef::new(b"fOo", b"BaR"),
         ];
 
+        let lookups: Vec<encoder::StaticLookup> = headers_in
+            .iter()
+            .map(|h| encoder::lookup_static(h))
+            .collect();
+
         let mut enc = Encoder::new();
-        assert_eq!(enc.encode(&headers_in, &mut encoded), Ok(35));
+        assert_eq!(enc.encode(&headers_in, &lookups, &mut encoded), Ok(35));
 
         let mut dec = Decoder::new();
         let headers_out = dec.decode(&mut encoded, std::u64::MAX).unwrap();
 
         assert_eq!(headers_expected, headers_out);
     }
+
+    #[test]
+    fn lower_ascii_range() {
+        let mut encoded = [0u8; 100];
+
+        // Indexed name with literal value
+        let headers = vec![crate::h3::Header::new(b"location", b"															")];
+
+        let lookups: Vec<encoder::StaticLookup> =
+            headers.iter().map(|h| encoder::lookup_static(h)).collect();
+
+        let mut enc = Encoder::new();
+        assert_eq!(enc.encode(&headers, &lookups, &mut encoded), Ok(49));
+
+        // Literal name and value
+        let headers = vec![crate::h3::Header::new(b"a", b"")];
```

Comment:
note: this looks like an empty string, but it is actually the <RS> character `` I thought Github fixed that after https://www.bleepingcomputer.com/news/security/invisible-characters-could-be-hiding-backdoors-in-your-javascript-code/ but guess not